tests/python/pants_test/releases/reversion_integration_test.py

```diff
@@ -41,7 +41,10 @@ def test_run(self):
                 dest_dir,
                 output_version,
             ]
-            self.assert_success(self.run_pants(command))
+
+            # NB: Pulling in the real BUILD file requires too many dependencies.
+            with self.temporary_file_content("src/python/pants/BUILD", b"python_library()"):
+                self.assert_success(self.run_pants(command))
```

Comment:
I think you want to update this:

https://github.com/pantsbuild/pants/blob/0d4feff1a6ae0bd8a05fb417e4c2d26849beca14/src/python/pants/BUILD#L60-L74

Generally, I'd love to delete that target.

Reply:
See the comment above this line.

Reply:
Yes, I saw it. You can now delete `util/BUILD`, can't you?

Reply:
No, it pulls in an `__init__.py`. Technically we could create _nearly_ all of the support files inside the test, but... not _actually_ all of them. So I don't see a way to get rid of this target. Given that, I pulled the minimum into the test.

Reply:
Okay, that's fine. Thanks. We can likely refactor that script anyways to be self-contained. It looks like we're only using `dirutil` for convenience methods that can be superseded by `pathlib.Path`, for example.